test/cutting/cut_finding/test_circuit_interfaces.py

```diff
@@ -68,7 +68,7 @@ def test_circuit_conversion(self):
         assert max_wire_cuts_gamma(7) == 2
 
         # Assign by hand a different qubit mapping by specifiying init_qubit_names.
-        circuit_converted = SimpleGateList(trial_circuit, ["q0", "q1"])
+        circuit_converted = SimpleGateList(trial_circuit, ("q0", "q1"))
```

Comment:
I think this test could stay as it was, actually.  It's okay to pass a `list`; it's just not okay when the default function argument is a `list`.

Reply:
Yes but I just figured this would be "more" consistent?

Reply:
Your call though.